plugin-hrm-form/src/states/storeNamespaces.ts

```diff
@@ -29,3 +29,4 @@ export const conferencingBase = 'conferencing';
 export const caseMergingBannersBase = 'caseMergingBanners';
 export const profileBase = 'profile';
 export const switchboardBase = 'switchboard';
+export const teamsviewBase = 'teamsview';
```

Comment:
```suggestion
export const teamsViewBase = 'teamsView';
```

This will drive me crazy 🤪